src/plugins/intel_gpu/src/graph/loop.cpp

```diff
@@ -803,8 +803,8 @@ void loop_inst::concatenated_memory_mapping::slice_mem(const int64_t num_iterati
     char* concat_data = reinterpret_cast<char*>(concatenated_mem->lock(stream, cldnn::mem_lock_type::read));
 
     auto concate_layout = concatenated_mem->get_layout();
-    auto trait = format::traits(concate_layout.format);
-    if (format::is_blocked(concate_layout.format) || concate_layout.data_padding) {
+    auto dims = concat_mem_shape.size();
+    if (dims == 1 || concate_layout.data_padding) {
```

Comment:
how about narrow the condition of using new slice calculation through checking default format with multi dimensions(dims > 1) to use new calculation only for default format.

Reply:
Applied fix as we discussed off-line